Add time stepper system updates (#24404)
idaholab/moose: modules/doc/content/newsletter/2023/2023_05.md

- [ElementCentroidPositions.md]
- [ElementGroupCentroidPositions.md]

> ### TimeStepper System

Reviewer: ```suggestion
### Possibility to combine TimeSteppers
```

> The time stepper system in MOOSE has been improved. We introduce a new [TimeSteppers](TimeSteppers/index.md) block to replace the current `[TimeStepper]` block. The new `[TimeSteppers]` keeps the features of `[TimeStepper]` and also supports the composed time step size with multiple time steps inputs. The users can still use `[TimeStepper]`, but in the future this block will be deprecated.

Reviewer 2: ```suggestion
The time stepper system in MOOSE has been improved. A new [`[TimeSteppers]`](TimeSteppers/index.md) block has been introduced to replace the functionality of the current `[TimeStepper]` block. The new system keeps the features of `[TimeStepper]` and also supports the composed time step size with multiple time step inputs. Users can still use `[TimeStepper]` for now, but in the future, this block will be deprecated.
```

> The time stepper system in MOOSE has been improved. We introduce a new [TimeSteppers](TimeSteppers/index.md) block to replace the current `[TimeStepper]` block. The new `[TimeSteppers]` keeps the features of `[TimeStepper]` and also supports the composed time step size with multiple time steps inputs. The users can still use `[TimeStepper]`, but in the future this block will be deprecated.

Reviewer: ```suggestion
The time stepper system in MOOSE has been improved. We introduce a new [TimeSteppers](TimeSteppers/index.md) block to replace the current `[TimeStepper]` block. The new `[TimeSteppers]` keeps the features of `[TimeStepper]` and also supports composing multiple time steppers to compute the time step size. The users can still use `[TimeStepper]`, but in the future this block will be deprecated.
```

> When more than one time steppers are provided, the time stepper system will automatically select the minimum time step among all the input time steppers as the output time step size. The time stepper system also provides a [!param](/Executioner/TimeSteppers/lower_bound) to further control the time stepper size (e.g. enable growth driven by a single time stepper). This system is controllable via [Controls](syntax/Controls/index.md) block. The user can turn on/off the time steppers to control the usage of time steppers like make time stepper(s) only active [at certain time period](TimePeriod.md).

Reviewer: ```suggestion
When more than one time stepper is provided, the time stepper system will automatically compute the minimum time step among all the input time steppers as the time step size to use for that application. The time stepper system also provides a [!param](/Executioner/TimeSteppers/lower_bound) to further control the time stepper size (e.g. enable growth driven by a single time stepper). This system is controllable via [Controls](syntax/Controls/index.md) block. The user can turn on/off the time steppers to control the usage of time steppers like make time stepper(s) only active [at certain time period](TimePeriod.md).
```

> When more than one time steppers are provided, the time stepper system will automatically select the minimum time step among all the input time steppers as the output time step size. The time stepper system also provides a [!param](/Executioner/TimeSteppers/lower_bound) to further control the time stepper size (e.g. enable growth driven by a single time stepper). This system is controllable via [Controls](syntax/Controls/index.md) block. The user can turn on/off the time steppers to control the usage of time steppers like make time stepper(s) only active [at certain time period](TimePeriod.md).

Reviewer 2: ```suggestion
When more than one time stepper is provided, the new time stepper system will automatically select the minimum time step among all of the input time steppers as the output time step size. The new time stepper system also provides a [!param](/Executioner/TimeSteppers/lower_bound) parameter to further control the time stepper size (e.g. enable growth driven by a single time stepper). This system is controllable via the [Controls](syntax/Controls/index.md) system. The user can turn on/off the provided time steppers to control the usage of time steppers; for example, one could make time stepper(s) only active [during a certain time period](TimePeriod.md).
```
